setup.py

```diff
@@ -3,16 +3,16 @@
 # TODO: if py gets upgrade to >=1.6,
 #       remove _width_of_current_line in terminal.py
 INSTALL_REQUIRES = [
-    "py>=1.5.0",
-    "packaging",
+    "atomicwrites>=1.0",
     "attrs>=17.4.0",
     "more-itertools>=4.0.0",
-    "atomicwrites>=1.0",
-    'pathlib2>=2.2.0;python_version<"3.6"',
-    'colorama;sys_platform=="win32"',
+    "packaging",
     "pluggy>=0.12,<1.0",
-    'importlib-metadata>=0.12;python_version<"3.8"',
+    "py>=1.5.0",
```

Comment:
It looks like the diff here is moving many more packages around than we're actually changing - any specific reason for that?

Plus the quotes make me think that `black` is not being applied to `setup.py`, which means we may want to check how autoformatting is configured...

Reply:
See first commit - sorts it.
Not sure if it's worth it - I'd rather leave it to avoid diff churn.

Reply:
Oh, it's not even adding to it?

Reply:
I don't have a strong opinion on sorting vs minimal diff, but if we're sorting it I'd go alphabetically *without* dividing by platform-specific requirements vs not.

Reply:
This always bothered me a bit because I think lists where the order doesn't matter should be sorted alphabetically in order to avoid conflicts.

I will remove that commit and rebase on `features`.